argon2-jvm-nolibs/src/main/java/de/mkammerer/argon2/Argon2Version.java

```diff
@@ -1,19 +1,28 @@
 package de.mkammerer.argon2;
 
+import de.mkammerer.argon2.jna.Argon2_version;
+
 /**
  * Version of the Argon2 algorithm.
  */
 public enum Argon2Version {
     V10(0x10),
-    V13(0x13);
+    V13(0x13),
+    NUMBER(V13.version);
```

Comment:
What is number? The default version?

Reply:
Yes it's more or less the current version.
My goal was to match the structure of `Argon2Version` to its C `enum` here:
https://github.com/P-H-C/phc-winner-argon2/blob/16d3df698db2486dde480b09a732bf9bf48599f9/include/argon2.h#L228

Reply:
Ah, i see. I would rename that to `DEFAULT_VERSION`.

Reply:
Alright will be done!